src/node_i18n.cc

```diff
@@ -170,22 +170,21 @@ MaybeLocal<Object> Transcode(Environment* env,
   return ret;
 }
 
-MaybeLocal<Object> TranscodeToUcs2(Environment* env,
-                                   const char* fromEncoding,
-                                   const char* toEncoding,
-                                   const char* source,
-                                   const size_t source_length,
-                                   UErrorCode* status) {
-  *status = U_ZERO_ERROR;
-  MaybeLocal<Object> ret;
+MaybeLocal<Object> TranscodeLatin1ToUcs2(Environment* env,
+                                         const char* fromEncoding,
+                                         const char* toEncoding,
+                                         const char* source,
+                                         const size_t source_length,
+                                         UErrorCode* status) {
   MaybeStackBuffer<UChar> destbuf(source_length);
-  Converter from(fromEncoding);
-  const size_t length_in_chars = source_length * sizeof(UChar);
-  ucnv_toUChars(from.conv(), *destbuf, length_in_chars,
-                source, source_length, status);
-  if (U_SUCCESS(*status))
-    ret = ToBufferEndian(env, &destbuf);
-  return ret;
+  auto actual_length =
+      simdutf::convert_latin1_to_utf16(source, source_length, destbuf.out());
+  if (actual_length == 0) {
+    *status = U_INVALID_CHAR_FOUND;
+    return {};
+  }
+
+  return ToBufferEndian(env, &destbuf);
```

Comment:
Ditto -- `simdutf::convert_latin1_to_utf16` already handles endianness:
https://github.com/simdutf/simdutf/blob/f058d353140f27257d68a5f45ab9ce35b2ed756c/src/implementation.cpp#L1061-L1067

Reply:
Without this line, it still fails.